internal/webhook/kueue/register.go

```diff
@@ -0,0 +1,19 @@
+//go:build !nowebhook
+
+package kueue
+
+import (
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+// RegisterWebhooks registers the webhooks for kueue.
+func RegisterWebhooks(mgr ctrl.Manager) error {
+	if err := (&Validator{
+		Client: mgr.GetClient(),
```

Comment:
We should use an uncached client here, as the cache may be out of date with the actual cluster state.

Reply:
@ugiordan this is true also for the other webhooks, maybe as a follow up PR for https://github.com/opendatahub-io/opendatahub-operator/pull/1987

Reply:
i am using mgr.GetAPIReader() becuase webhook is only reading.

Reply:
In the Kueue webhook handler, I am using mgr.GetAPIReader() of type client.Reader. For now, the Kueue handler doesn't have any defaulting.

Reply:
@GowthamShanmugam yeah, we can use `mgr.GetAPIReader()` as well